rke/k8s_rke_system_images.go

```diff
@@ -8902,6 +8902,50 @@ func loadK8sRKESystemImages() map[string]v3.RKESystemImages {
 			WindowsPodInfraContainer:  "rancher/mirrored-pause:3.7",
 			Nodelocal:                 "rancher/mirrored-k8s-dns-node-cache:1.22.10",
 		},
+		// Enabled in Rancher v2.7.4
+		"v1.26.4-rancher2-1": {
+			Etcd:                      "rancher/mirrored-coreos-etcd:v3.5.6",
+			Kubernetes:                "rancher/hyperkube:v1.26.4-rancher2",
+			Alpine:                    "rancher/rke-tools:v0.1.89",
+			NginxProxy:                "rancher/rke-tools:v0.1.89",
+			CertDownloader:            "rancher/rke-tools:v0.1.89",
+			KubernetesServicesSidecar: "rancher/rke-tools:v0.1.89",
+			KubeDNS:                   "rancher/mirrored-k8s-dns-kube-dns:1.22.8",
+			DNSmasq:                   "rancher/mirrored-k8s-dns-dnsmasq-nanny:1.22.8",
+			KubeDNSSidecar:            "rancher/mirrored-k8s-dns-sidecar:1.22.8",
+			KubeDNSAutoscaler:         "rancher/mirrored-cluster-proportional-autoscaler:1.8.6",
+			Flannel:                   "rancher/mirrored-flannelcni-flannel:v0.21.4",
+			FlannelCNI:                "rancher/flannel-cni:v0.3.0-rancher7",
+			CalicoNode:                "rancher/mirrored-calico-node:v3.25.0",
+			CalicoCNI:                 "rancher/calico-cni:v3.25.0-rancher1",
+			CalicoControllers:         "rancher/mirrored-calico-kube-controllers:v3.25.0",
+			CalicoCtl:                 "rancher/mirrored-calico-ctl:v3.25.0",
+			CalicoFlexVol:             "rancher/mirrored-calico-pod2daemon-flexvol:v3.25.0",
+			CanalNode:                 "rancher/mirrored-calico-node:v3.25.0",
+			CanalCNI:                  "rancher/calico-cni:v3.25.0-rancher1",
+			CanalControllers:          "rancher/mirrored-calico-kube-controllers:v3.25.0",
+			CanalFlannel:              "rancher/mirrored-flannelcni-flannel:v0.21.4",
+			CanalFlexVol:              "rancher/mirrored-calico-pod2daemon-flexvol:v3.25.0",
+			WeaveNode:                 "weaveworks/weave-kube:2.8.1",
+			WeaveCNI:                  "weaveworks/weave-npc:2.8.1",
+			AciCniDeployContainer:     "noiro/cnideploy:5.2.3.6.1d150da",
+			AciHostContainer:          "noiro/aci-containers-host:5.2.3.6.1d150da",
+			AciOpflexContainer:        "noiro/opflex:5.2.3.6.1d150da",
+			AciMcastContainer:         "noiro/opflex:5.2.3.6.1d150da",
+			AciOpenvSwitchContainer:   "noiro/openvswitch:5.2.3.6.1d150da",
+			AciControllerContainer:    "noiro/aci-containers-controller:5.2.3.6.1d150da",
+			AciGbpServerContainer:     "noiro/gbp-server:5.2.3.6.1d150da",
+			AciOpflexServerContainer:  "noiro/opflex-server:5.2.3.6.1d150da",
+			PodInfraContainer:         "rancher/mirrored-pause:3.7",
+			Ingress:                   "rancher/nginx-ingress-controller:nginx-1.5.1-rancher2",
```

Comment:
New nginx releases available https://github.com/kubernetes/ingress-nginx/releases, note https://github.com/kubernetes/ingress-nginx/releases/tag/controller-v1.7.0 drops support for v1.23 so let's use https://github.com/kubernetes/ingress-nginx/releases/tag/controller-v1.6.4 which supports v1.23 and v1.26 both https://github.com/kubernetes/ingress-nginx#supported-versions-table